src/AutoNumeric.js

```diff
@@ -6758,6 +6758,13 @@ To solve that, you'd need to either set \`decimalPlacesRawValue\` to \`null\`, o
                 }
             }
         }
+        else if ((this.eventKey === AutoNumericEnum.keyName.Y || this.eventKey === AutoNumericEnum.keyName.y) && e.ctrlKey) {
+            // Redo
+            e.preventDefault();
+            this._historyTableRedo();
+            this._triggerEvent(AutoNumeric.events.native.input, e.target);
+            return;
```

Comment:
Could you please add an empty line before the `return;`?

Reply:
Done - I force pushed on the same branch.